src/System.Net.Mail/tests/Functional/AlternateViewCollectionTest.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+//
+// AlternateViewCollectionTest.cs - NUnit Test Cases for System.Net.MailAddress.AlternateViewCollection
+//
+// Authors:
+//   John Luke (john.luke@gmail.com)
+//
+// (C) 2005 John Luke
```

Comment:
@richlander, why is that?  What's special about a comment at the top of the file that we need to keep the whole thing, even parts that aren't specific to a copyright or permissions?  The comment is simple wrong at this point (e.g. these aren't NUnit tests).